torch/_inductor/lowering.py

```diff
@@ -164,8 +164,6 @@ def wrapped(*args, **kwargs):
             args = args[0]
         # Only look at args that are Tensors
         indices = [i for i, x in enumerate(args) if isinstance(x, TensorBox)]
-        # kwargs tensors not supported yet
```

Comment:
what would happen if regular (non-fallback) kernel has tensor kwargs?

Reply:
Then we need to use pytree here too. Since we haven't hit the case yet, mind if I just alter the assert condition here to exclude the fallback ops, and come back to update when we hit the case later?

Reply:
Yes, absolutely